volumes/miovision/sql/function/function-clear-api_log.sql

```diff
@@ -0,0 +1,31 @@
+﻿CREATE OR REPLACE FUNCTION miovision_api.clear_api_log(
+    _start_date date,
+    _end_date date,
+    intersection integer DEFAULT NULL)
```

Comment:
maybe good if it takes an array, in case we want to get rid of a couple ones that fails (? 
just a thought 🧠